src/orders/components/OrderProductsCardElements/OrderProductsTableRow.tsx

```diff
@@ -89,11 +96,25 @@ const TableLine: React.FC<TableLineProps> = ({
     ? quantity - quantityFulfilled
     : quantity;
 
+  const isDeleted = !line.orderLine.variant;
+
   return (
     <TableRow className={classes.clickableRow} hover key={line.id}>
       <TableCellAvatar
         className={classes.colName}
         thumbnail={maybe(() => line.orderLine.thumbnail.url)}
+        badge={
+          isDeleted && (
+            <StatusBadge
+              variant={isFulfilled ? "warning" : "error"}
+              message={
+                isFulfilled
+                  ? intl.formatMessage(messages.fulfilledVariantDeleted)
+                  : intl.formatMessage(messages.unfulfilledVariantDeleted)
+              }
+            />
+          )
+        }
```

Comment:
I'm not sure if having `isFulfilled ? <StatusBadge ... /> : <StatusBadge .... />` wouldn't be more readable

Reply:
will do